src/api/routes/stx-supply.ts

```diff
@@ -70,7 +74,11 @@ export const StxSupplyRoutes: FastifyPluginAsync<
                   'String quoted decimal number of the percentage of STX that have unlocked',
               }),
               total_stx: Type.String({
-                description: 'String quoted decimal number of the total possible number of STX',
+                description: 'String quoted decimal number of the total circulating number of STX',
```

Comment:
Maybe qualify "at the input block height" or something like that?

Reply:
Added